### PR TITLE
update to MOI 1 and delete Manifest.toml

### PR DESCRIPTION
**Pull request by chriscoey**
replaces #43

### CODE REVIEW

**Comment by chriscoey** [comment]
@joehuchette I don't see any CI, but tests pass locally.

File: Project.toml [hunk 1]
```diff
@@ -10,9 +10,6 @@ MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 
 [compat]
-Cbc = "≥ 0.6.0"
-JuMP = "~0.22"
-MathOptInterface = "~0.10"
 julia = "1"
```
Comment (blegat) [inline Project.toml]: You should keep a compat for JuMP and MOI. Otherwise, releasing won't be possible

Comment (chriscoey) [reply Project.toml]: thanks, realized that the hard way with Hypatia last night.

**Comment by joehuchette** [comment]
I cannot check manually right now, but looks reasonable. Merging, thanks @chriscoey

**Comment by chriscoey** [comment]
thanks @joehuchette. #43 can be closed now. also, would you mind tagging the package?

**Comment by joehuchette** [comment]
Just tagged v0.4 on Github. I’m on mobile for the week so I can’t do anything further that requires Julia. Feel free to jump in, and I can add you as collaborator on the repo if needed.

**Pull request merged**